### PR TITLE
Implement responsive desktop version to my portfolio

### PR DESCRIPTION
**Pull request by rskinnerc**
#### In this PR I have:
- Layout the different sections of my portfolio using _grid_ and _flex_ implementing _media queries_.
- Add the desktop compatible background SVG images.
- Add the menu list to the header.
- Refactor some minor HTML elements to make them responsive.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -323,3 +323,152 @@ footer {
...
+    border: 1px solid #dfe1e6;
+    position: relative;
+    bottom: 56px;
+  }
+
+  footer .footer-social .social-media ~ * {
+    display: none;
+  }
+}
+
```
Comment (BenMukebo) [inline styles.css:468]: - [ ] I really appreciate your design, you did great work so far, however, please implement these few changes and you will be good to go, see this image below, your paragraph text alignment is not the same as on the original design.
adjust your

- [ IT SHOULD LOOK LIKE THIS INSTEAD]
fig

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @rskinnerc, **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!



## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rskinnerc** [comment]
Dear @BenMukebo thank you for your feedback.

I have pushed a slightly modified version of the footer height, which made the paragraph more centered in the vertical axe. However, please note at the same width than the Figma template (1440px) it resembles almost the same position/appearance. However if you downsize the viewport, and as it is a responsive design, it begins to shrink, which I think is the desired behavior (Which we are not able to test in the template, but I'm assuming it is the expected, otherwise I'd have to make the div a fixed width so it stays the same across all viewport widths (until it reaches mobile). 

Thank you for your work 🙏

### A screenshot of my final outcome with the viewport's width set to 1440px (as the template)
image

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @rskinnerc,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**